### PR TITLE
ci(build-push): add build-push image workflow

### PR DESCRIPTION
**Pull request by ar-bella-jeong**
- add build-push image workflow

### CODE REVIEW

**Review by wqld — DISMISSED** [review]
LGTM :+1:

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,17 +12,24 @@ jobs:
...
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v2
+
       - name: Generate tag name
         id: tag
         run: |
-          IMAGE_TAG="ghcr.io/grasse-oss/cron-set-controller:"${GITHUB_REF#refs/tags/v}"
+          # IMAGE_TAG="ghcr.io/grasse-oss/cron-set-controller:"${GITHUB_REF#refs/tags/v}"
+          IMAGE_TAG="ghcr.io/grasse-oss/cron-set-controller:0.0.2"
```
Comment (wqld) [inline .github/workflows/build.yml]: Is there a reason why the tag should be fixed at 0.0.2?

Comment (ar-bella-jeong) [reply .github/workflows/build.yml]: It was temporary added for testing. now it isn't exist.

**Review by wqld — APPROVED** [review]
LGTM :+1:

**Pull request merged**